projects/cdk/schematics/ng-update/steps/migrate-taiga-proprietary-icons.ts

```diff
@@ -45,39 +44,43 @@ export function migrateTaigaProprietaryIcons(options: TuiSchema): Rule {
                       return;
                   }
 
-                  const project = getProject(options, workspace);
+                  const projects = getProjects(options, workspace);
 
-                  if (!project) {
+                  if (!projects.length) {
                       context.logger.warn(
                           `[WARNING]: Target project not found. ${MANUAL_MIGRATION_TIPS}`,
                       );
                       return;
                   }
 
-                  let targetOptions;
-
-                  try {
-                      targetOptions = getProjectTargetOptions(project, 'build');
-                  } catch {
-                      context.logger.warn(
-                          `[WARNING]: No buildable project was found. ${MANUAL_MIGRATION_TIPS}`,
-                      );
-                      return;
-                  }
-
-                  if (Array.isArray(targetOptions?.assets)) {
-                      const tdsSrc = '@taiga-ui/proprietary-tds-icons/src';
-                      const hasIcons = (targetOptions.assets as Asset[]).find(asset => {
-                          return typeof asset === 'string'
-                              ? asset.includes(tdsSrc)
-                              : asset?.input?.includes(tdsSrc);
-                      });
+                  for (let project of projects) {
+                      let targetOptions;
 
-                      if (hasIcons) {
+                      try {
+                          targetOptions = getProjectTargetOptions(project, 'build');
+                      } catch {
+                          context.logger.warn(
+                              `[WARNING]: No buildable project was found. ${MANUAL_MIGRATION_TIPS}`,
```

Comment:
**nit**

Could we print projectName here?
Something like 
```suggestion
                              `[WARNING]: ${PROJECT_NAME} No buildable project was found. ${MANUAL_MIGRATION_TIPS}`,
```

If user has 5 projects with no `build`-target, it will get 5 identical logs 🤔

Reply:
will resolved in another pr